bring/src/main/java/com/bobocode/petros/bring/registry/DefaultBeanDefinitionRegistry.java

```diff
@@ -72,6 +74,15 @@ public BeanDefinition getBeanDefinition(final String beanName) {
         return beanDefinitions.get(beanName);
     }
 
+    @Override
+    public Collection<BeanDefinition> getAllBeanDefinitions() {
+        Collection<BeanDefinition> definitionCollection = beanDefinitions.values();
+        if (definitionCollection.isEmpty()) {
+            return new ArrayList<>();
```

Comment:
new ArrayList<>() could be replaced with Collections.emptyList() which is immutable.